show/main.py

```diff
@@ -966,6 +966,28 @@ def counters(ctx, verbose, period, printall):
 
         run_command(cmd, display_cmd=verbose)
 
+# 'errors' subcommand ("show interfaces counters errors")
+@counters.command()
+@click.option('-p', '--period')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def errors(verbose, period):
+    """Show interface counters errors"""
+    cmd = "portstat -e"
+    if period is not None:
+        cmd += " -p {}".format(period)
+    run_command(cmd, display_cmd=verbose)
+
+# 'rate' subcommand ("show interfaces counters rate")
+@counters.command()
+@click.option('-p', '--period')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def rate(verbose, period):
```

Comment:
'errors' is plural, whereas 'rate' is singular. I think we should pick one form and stick with it. I'm leaning toward plural, since there are multiple types of errors and multiple types of rates. Plus, our CLI accepts abbreviations, so the singular forms should still work unless similar subcommand names are ever added to the group.

Reply:
updated.